src/main/java/duke/command/Add.java

```diff
@@ -0,0 +1,40 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.task.TaskList;
+import duke.ui.Ui;
+
+/**
+ * Encapsulates an Add command that deals with adding task to the task list.
+ *
+ * @author Zhi Bin
+ * @version Duke Level 9
+ */
+public class Add extends DukeCommand {
```

Comment:
To show that this class extends DukeCommand perhaps name it something like AddCommand to make it clear that this is a command.